game/test/ecs/components/HealthComponentTest.java

```diff
@@ -0,0 +1,109 @@
+package ecs.components;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.times;
+
+import ecs.damage.Damage;
+import ecs.damage.DamageType;
+import ecs.entities.Entity;
+import graphic.Animation;
+import mydungeon.ECS;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class HealthComponentTest {
+
+    @Test
+    public void getHit() {
+        ECS.entities.clear();
+        Entity entity = new Entity();
+        HealthComponent hc = new HealthComponent(entity);
+        Damage dmg = new Damage(3, DamageType.FIRE);
+        hc.getHit(dmg);
+        assertTrue(hc.getDamageList().contains(dmg));
```

Comment:
```suggestion
        assertTrue(hc.getDamage(DamageType.FIRE));
```

Reply:
Sollte hier nicht auch auf den konkreten Damage überprüft werden, also ob der in `hc` gespeicherte Damage dann den Wert 3 hat?!